adm/src/commands/blockstore.rs

```diff
@@ -202,11 +207,13 @@ fn run_show_command<'a>(args: &ArgMatches<'a>) -> Result<(), CliError> {
     }
     .map_err(|err| CliError::ArgumentError(format!("Error getting block: {}", err)))?;
 
-    let block_wrapper = BlockWrapper::try_from(block)
-        .map_err(|err| CliError::EnvironmentError(format!("{}", err)))?;
+    let block_wrapper = BlockWrapper::try_from(block).map_err(|err| {
+        CliError::EnvironmentError(format!("failed to great block wrapper: {}", err))
```

Comment:
Should this say "get block wrapper"

Reply:
ha - actually, it should say "create".  Fixed.